test/utility/fragments.ts

```diff
@@ -81,6 +81,14 @@ export const org = gql`
             }
           }
         }
+        defaultMarketingRegion {
```

Comment:
I wouldn't really something we should consider when looking for orgs

Reply:
Are you saying this is not needed here?
"I wouldn't really think this is something we should consider when looking for orgs"

Reply:
Yeah not needed here, along with a lot of these other location fields FWIW